### ISSUE TITLE
all_vs_all warning

### ISSUE THREAD

**Issue by ZilongXu** [patrickbryant1/SpeedPPI#3]
Hi, the following problems occurred during using "create_ppi_all_vs_all.sh" for test file :
/mnt/label/home/wanghao/software/SpeedPPI-master/src/alphafold/model/mapping.py:49: FutureWarning: jax.tree_flatten is deprecated, and will be removed in a future release. Use jax.tree_util.tree_flatten instead.
  values_tree_def = jax.tree_flatten(values)[1]
/mnt/label/home/wanghao/software/SpeedPPI-master/src/alphafold/model/mapping.py:53: FutureWarning: jax.tree_unflatten is deprecated, and will be removed in a future release. Use jax.tree_util.tree_unflatten instead.
  return jax.tree_unflatten(values_tree_def, flat_axes)
/mnt/label/home/wanghao/software/SpeedPPI-master/src/alphafold/model/mapping.py:124: FutureWarning: jax.tree_flatten is deprecated, and will be removed in a future release. Use jax.tree_util.tree_flatten instead.
  flat_sizes = jax.tree_flatten(in_sizes)[0]

**Comment by patrickbryant1** [comment]
Hi, this is only a warning and should have no impact on the predictions.

**Comment by ZilongXu** [comment]
OK. Thank you.
